tools/build/docker/s6/fix-attrs.d/01-lrr-dirs

```diff
@@ -0,0 +1,7 @@
+# second field in user is fallback uid
```

Comment:
it's just "the s6 way". personally I think it looks a bit nicer and easier to maintain, but it's currently not in use right now anyway (the COPY line in the Dockerfile is commented) due to the fact the startup script recursively chowns the user, but not the mode. is that done for a specific reason, by the way?

Reply:
oh, also the fix-attrs.d scripts run before the cont-init.d scripts, so "koyomi" won't have had its uid/gid changed to match the envvars before we can change the perms

Reply:
I think not changing the mode was simply an oversight so that'd be fine, but not matching the uid/gids to the envvars is a major problem. 😥

Reply:
Oh right the chmods in the entrypoint/now init script, well there are various reasons: 

-  `/home/koyomi/lanraragi` needs to be writable due to the shinobu lockfiles and filemap data being dumped there, but making it recursive would change perms on the content directory. Certainly got my lot of angry email about that..
- For the thumbnail/log/temp folder there's no real reason I can see at a glance so that could be recurse'd.

Reply:
gotcha, i'll leave the file there for now then unused and keep the manual chmod/owning in cont-init.d